coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,9 +207,9 @@ def get_executable():
             Returns the executable of this class.
 
             :return:
-                The executable name.
+                The full path to the executable file.
             """
-            return options['executable']
+            return shutil.which(options['executable'])
```

Comment:
@userzimmermann did not change and I thought he had a reason for it. But assigning the full path is definitely better, you're right^^ @arush0311 :+1: